sympy/geometry/tests/test_polygon.py

```diff
@@ -522,3 +522,12 @@ def test_second_moment_of_area():
     assert (I_xx - rectangle.second_moment_of_area(p)[0]) == 0
     assert (I_yy - rectangle.second_moment_of_area(p)[1]) == 0
     assert (I_xy - rectangle.second_moment_of_area(p)[2]) == 0
+
+
+def test_cut_section():
+    p = Polygon((-1, -1),(1, 2.5),(2, 1),(3, 2.5),(4, 2),(5, 3),(-1, 3))
+    l = Line((0, 0),(4.5, 3))
```

Comment:
Using `float` type for coordinates is causing the job to fail. You should write them in `Rational` instead:
```
p = Polygon((-1, -1),(1, S(5)/2),(2, 1),(3, S(5)/2),(4, 2),(5, 3),(-1, 3))
l = Line((0, 0),(S(9)/2, 3))
```